x/margin/keeper/msg_server.go

```diff
@@ -282,7 +286,7 @@ func (k Keeper) ForceCloseLong(ctx sdk.Context, msg *types.MsgForceClose) (*type
 	}
 
 	// check MTP health against threshold
-	forceCloseThreshold := k.GetForceCloseThreshold(ctx)
+	forceCloseThreshold := k.GetSafetyFactor(ctx)
```

Comment:
@timlind I don't we needed to do additional calculation along with SF to be able to use it as a force close threshold am I missing something?

Reply:
@timlind you can ignore that comment as @Rubz2021 confirmed SF should be used instead in-place of previous force close threshold, thanks